agpypeline/geoimage.py

```diff
@@ -205,6 +207,52 @@ def create_geotiff(pixels: np.ndarray, gps_bounds: tuple, out_path: str, srid: i
             output_raster.GetRasterBand(1).SetNoDataValue(nodata)
 
 
+def get_centroid_latlon(filename: str) -> ogr.Geometry:
+    """Returns the bounds and epsg of a geo-referenced image file
+    Arguments:
+        filename: the path to the file to get the centroid from
+    Returns:
+        Returns the centroid of the geometry loaded from the file in lat-lon coordinates
+        as a result of calling geometries.get_centroid_latlon()
+    Exceptions:
+        RuntimeError is raised if the image is not a geo referenced image with an EPSG code,
+        the EPSG code is not supported, or another problems occurs
+    """
+    epsg = get_epsg(filename)
+
+    poly = get_image_bounds(filename, epsg)
+    if poly is None:
+        msg = "File is not a geo-referenced image file: %s" % filename
+        logging.error(msg)
+        raise RuntimeError(msg)
+    if epsg is None:
+        msg = "EPSG is not found in image file: '%s'" % filename
+        logging.error(msg)
+        raise RuntimeError(msg)
+    # Convert the polygon to lat-lon
+    dest_spatial = osr.SpatialReference()
+    if int(osgeo.__version__[0]) >= 3:
+        # GDAL 3 changes axis order: https://github.com/OSGeo/gdal/issues/1546
+        # pylint: disable=no-member
+        dest_spatial.SetAxisMappingStrategy(osgeo.osr.OAMS_TRADITIONAL_GIS_ORDER)
+
+    if dest_spatial.ImportFromEPSG(int(LAT_LON_EPSG_CODE)) != ogr.OGRERR_NONE:
+        msg = "Failed to import EPSG %s for conversion to lat-lon" % str(LAT_LON_EPSG_CODE)
+        logging.error(msg)
+        raise RuntimeError(msg)
+    ref_sys = osr.SpatialReference()
+    ref_sys.ImportFromEPSG(int(epsg))  # an error would have been caught before this point
```

Comment:
> an error would have been caught before this point

It's not clear where else this might have been caught. Can you either update the comment, or, (better yet IMO), put the comparison to `ogr.OGERR_NONE` back?